hphp/runtime/ext/std/ext_std_file.cpp

```diff
@@ -193,7 +193,15 @@ static int accessSyscall(
         return -1;
       }
     }
-    return ::access(File::TranslatePathWithFileCache(path).data(), mode);
```

Comment:
I think this change will disable the StaticFileCache feature.

The cause of this bug is that FileUtil::canonicalize is used in File::TranslatePathWithFileCache and File::TranslatePathKeepRelative. So what's the purpose of FileUtil::canonicalize? It a path is canonicalized, the access() result may be different.